examples/base/land-sand-example.yaml

```diff
@@ -0,0 +1,23 @@
+---
+type: Feature
+geometry:
+  type: Polygon
+  coordinates: [[[34.4379197, 28.7592689], [34.4380109, 28.7595816], [34.4380173, 28.7601587], [34.4380538, 28.7606186],[34.4381540, 28.7609863], [34.4380735, 28.7612459], [34.4377346, 28.7608396], [34.4377158, 28.7606139],[34.4377105, 28.7605527], [34.4376300, 28.7604258], [34.4376407, 28.7603599], [34.4376488, 28.7602494],[34.4376863, 28.7599673], [34.4376675, 28.7597980], [34.4376890, 28.7596216], [34.4379197, 28.7592689]]]
+properties:
+  theme: base
+  type: land
+  class: sand
+  subClass: dune
```

Comment:
One of my more important comments from last week seems to have gotten clobbered, or at any rate I can't find it anymore.

So far the schema property "hierarchy" implied by the other layers is: `type > subType > class`. Here we have `type > class > subClass`.

There's an exception to this in `buildings` where we go immediately to `type > class` but I think we should strive to normalize this so that customers and consumers can look at and query a predictable set of columns rather than an *ad hoc* set that varies by theme and feature type.

My solution would be:

- Transportation: Promote `road.class` up to main feature properties.
- Buildings: Rename `class` to `subType`.
- Base: Rename `class` to `subType` and `subClass` to `class`.

Obviously this will be a bit less aligned with the Earth tables than previously, but Overture is a different product and we want to present a coherent, structured, internally consistent picture across all the themes and types.

Reply:
@vcschapp I mostly agree with you, but there's something about the hierarchy that feels off to me for some things, check this out: 

1. type=water, subType = 'manMade', class= 'pool' ✅ 
2. type=land, subType = 'grass', class='grass' ✅ 
3. type=landuse, subType = 'agriculture', class='farmland' ✅ 
4. type=building, subType = 'residential' ?, class=None ?? 

I think the class -> subType makes sense for the base theme, but the building classes (and this opens up some of the original buildings argument) is that `class` applies to the _use_ of the building, not a subType of the building. I could imagine `subType` being "shed" or "garage" or something, but `class` is about the current use.

Reply:
> I think the class -> subType makes sense for the base theme, but the building classes (and this opens up some of the original buildings argument) is that `class` applies to the *use* of the building, not a subType of the building.

Isn't this mostly a question of definitions?

The interesting thing is if you just take the words *type* and *class* in the abstract, I think most people would have a hard time differentiating between them. They're more or less synonyms. (I guess what I'm saying could be an argument for not allowing both `class` and `type`/`subType` in the top-level schema.)

I think if we define `subType` for `building` to mean current use, and make that clear, people would understand.